.github/workflows/linter_test.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       max-parallel: 3
       matrix:
-        lint: ['flake8', 'black', 'isort']
+        lint: ['flake8', 'black', 'isort', 'ufmt']
```

Comment:
Do you need black and isort now after ufmt?

Reply:
Good point, we shouldn't need them anymore. Let me remove them